### PR TITLE
shrug, simpleCommands, and time tests

### PR DESCRIPTION
**Pull request by twalton83**
- Created snapshots + tests for the `shrug.js`, `simpleCommands.js`, and `time.js` files. 
- The snapshots for shrug are a bit wonky, but I think it's due to escaping characters. I checked the output being returned with a console.log, and it doesn't look the same as the output being returned in the snapshots, even if it does match up properly when run through the bot and manually checked. I'll likely need some review here.
- Additionally, I had to make a few lines dedicated to checking that all of the permutations of the word "shrug" would trigger the regex. I wasn't sure if I needed to do the same in other areas, and am looking to discuss how to improve this piece. checking each one would result in thousands of tests for this one file, so I am not sure if that was necessary if we already checked if the permutations all worked and we only need to know if ONE permutation triggered the regex for sections such as "command should be its own word/group - no leading or trailing characters", or "command can be anywhere in the string".

### CODE REVIEW

File: botCommands/shrug.test.js [hunk 1]
```diff
@@ -1,7 +1,56 @@
 const command = require('./shrug')
 
-describe('', ()=>{
-    xit('',()=> {
+describe('/shrug', () => {
+  const permutations = "SHRUG HSRUG RSHUG SRHUG HRSUG RHSUG UHSRG HUSRG SUHRG USHRG HSURG SHURG SRUHG RSUHG USRHG SURHG RUSHG URSHG URHSG RUHSG HURSG UHRSG RHUSG HRUSG GRUSH RGUSH UGRSH GURSH RUGSH URGSH SRGUH RSGUH GSRUH SGRUH RGSUH GRSUH GUSRH UGSRH SGURH GSURH USGRH SUGRH SURGH USRGH RSUGH SRUGH URSGH RUSGH HUSGR UHSGR SHUGR HSUGR USHGR SUHGR GUHSR UGHSR HGUSR GHUSR UHGSR HUGSR HSGUR SHGUR GHSUR HGSUR SGHUR GSHUR GSUHR SGUHR UGSHR GUSHR SUGHR USGHR RSGHU SRGHU GRSHU RGSHU SGRHU GSRHU HSRGU SHRGU RHSGU HRSGU SRHGU RSHGU RGHSU GRHSU HRGSU RHGSU GHRSU HGRSU HGSRU GHSRU SHGRU HSGRU GSHRU SGHRU UGHRS GUHRS HUGRS UHGRS GHURS HGURS RGUHS GRUHS URGHS RUGHS GURHS UGRHS UHRGS HURGS RUHGS URHGS HRUGS RHUGS RHGUS HRGUS GRHUS RGHUS HGRUS GHRUS"
```
Comment (I3uckwheat) [inline botCommands/shrug.test.js:4]: Stuff like this is just fine for a unit test. The idea is to add stuff as we go. 

the /shrug command is a hard one to test, this all looks fine to me.

File: botCommands/shrug.test.js [hunk 2]
```diff
@@ -1,7 +1,56 @@
...
+      ['@odin-bot / shrug'],
+      ['/shru&g'],
+      ['/shr^ug'],
+      ['/shrug!'],
+      ['@odin-bot/ shrug'],
+      ['https://shrug.com']
+    ])("'%s' does not trigger the callback", (string) => {
+      expect(command.regex.test(string)).toBeFalsy()
+    })
+    it.each([
```
Comment (I3uckwheat) [inline botCommands/shrug.test.js:32]: Consider newlines here to make it easier to parse

File: botCommands/simpleCommands.test.js [hunk 3]
```diff
@@ -1,7 +1,779 @@
...
+      ['it\'s about/smart'],
+      ['/smartisanillusion'],
+      ['/smart/'],
+      ['/smart*'],
+      ['/smart...']
+    ])("'%s' - command should be its own word/group - no leading or trailing characters", (string) => {
+      expect(commands.smart.regex.test(string)).toBeFalsy()
+    })
+  })
+  describe('callback', () => {
```
Comment (I3uckwheat) [inline botCommands/simpleCommands.test.js:106]: New line above 106 would help with readability.

**Review by I3uckwheat — CHANGES_REQUESTED** [review]
Looks pretty good other than a few formatting things that should be fixed 👍

**Comment by twalton83** [comment]
@I3uckwheat I unintentionally added some other spacing changes I had made to the commit.

**Review by I3uckwheat — APPROVED** [review]

**Comment by I3uckwheat** [comment]
@twalton83 We'll get it merged then worry about adding eslint and prettier after the fact.

**Comment by twalton83** [comment]
@I3uckwheat Thanks!

**Pull request merged**